### ISSUE TITLE
OSD in an iframe fails to end drag

### ISSUE THREAD

**Issue by waltsatan** [openseadragon/openseadragon#579]
I have an OpenSeadragon viewer inside an iframe.  I'm able to pick up the mouseup outside the iframe by attaching a `$(document).add(parent.document).mouseup()` listener, but how can I forward this to the viewer?
I've tried:

``` javascript
viewer.raiseEvent('canvas-drag-end');
viewer.raiseEvent('mouseup');
$( "#openseadragon" ).trigger( "mouseup" );
```

... to no avail.

Thanks!
Alan

**Comment by msalsbery** [comment]
A fix for that is being worked on right now. Should be in soon, hopefully in the 1.2.1 release early next week.

**Comment by waltsatan** [comment]
Perfect - thanks for the update.

**Comment by EmilyReif** [comment]
What's the update on this?

Thanks!

**Comment by iangilman** [comment]
@msalsbery ?

**Comment by avandecreme** [comment]
Was fixed in #790 if the iframe and the containing window are on the same domain.
